src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentCost = 40;
+  const totalRentPrice = days * rentCost;
+  const longTermRent = 7;
+  const longTermDiscount = 50;
+  const smallTermRent = 3;
+  const smallTermDiscount = 20;
+
+  if (days < smallTermRent) {
+    return totalRentPrice;
+  }
+
+  if (days >= smallTermRent && days <= 6) {
+    return totalRentPrice - smallTermDiscount;
+  }
+
+  if (days >= longTermRent) {
+    return totalRentPrice - longTermDiscount;
```

Comment:
You can remove the `if` statement here. Since all previous conditions return if they are true, you can simply return the calculation for the long-term discount without checking if `days >= longTermRent` again.